### ISSUE TITLE
Tracker emojis don't show on macOS Safari

### ISSUE THREAD

**Issue by trych** [open-nomie/nomie#35]
On Safari 12.1.2, macOS 10.13.6, all trackers look like this:

![20190827-095232_Screenshot_Safari](https://user-images.githubusercontent.com/9803905/63752001-b51eba80-c8b0-11e9-9312-98fb8135104a.png)

**Comment by trych** [comment]
No, emojis are generally available in my version of macOS and also in Safari, it must be a different problem. They also show on other pages in Safari (like Wikipedia). Which type of emoji I choose does not matter, even the most generic emoji is not diplayed.

Could someone else with Safari confirm this issue?

**Comment by trych** [comment]
Yes, I can see this. Maybe it's my not-up-to-date macOS version then? Feel free to close this issue. If I am the only one who has it, it does not matter. If someone else comes along, we could re-open it and then compare what my and their setup have in common.

Yes, I can change the emoji, but in the input field I also only see the missing symbol glyph.

**Comment by trych** [comment]
Good tag. 😆

**Comment by trych** [comment]
Oh and if we're already talking about emojis, one more UI suggestion related to that: When you add a tracker the white dot emoji ⚪️ is there by default. When I first saw this, I thought it was supposed to be a radio button that I had to click for some reason.

So, I would suggest to change this to something else, so that users understand right away that it is an emoji they are seeing and can edit not some random ui input element that they have to click.
